sdk/nestjs/src/DevCycleModule/RequestWithData.ts

```diff
@@ -0,0 +1,24 @@
+import { Request } from 'express'
```

Comment:
what if they aren't using express as the HTTP adapter? 

also there should be a way to get this type without importing it from the express package via inference from the Nest types. We shouldn't add a direct dependency on express with this package either

Reply:
Nest's docs suggest importing the request types from express, we could instead leave the request type as any object with the dvc_client and dvc_user properties defined

Reply:
yeah if the type isn't meant to be used externally to this package then just type it as any, better that than adding a dependency that needs to be installed just for internal TS typing benefit